gtsam/sfm/tests/testShonanAveraging.cpp

```diff
@@ -158,17 +158,17 @@ TEST(ShonanAveraging3, CheckWithEigen) {
   double lambda = kShonan.computeMinEigenValue(Qstar3);
 
   // Check Eigenvalue with slow Eigen version, converts matrix A to dense matrix!
-  const Matrix S = ShonanAveraging3::StiefelElementMatrix(Qstar3);
-  auto A = kShonan.computeA(S);
-  bool computeEigenvectors = false;
-  Eigen::EigenSolver<Matrix> eigenSolver(Matrix(A), computeEigenvectors);
-  auto lambdas = eigenSolver.eigenvalues().real();
-  double minEigenValue = lambdas(0);
-  for (int i = 1; i < lambdas.size(); i++)
-      minEigenValue = min(lambdas(i), minEigenValue);
+  // const Matrix S = ShonanAveraging3::StiefelElementMatrix(Qstar3);
```

Comment:
Please re-enable commented out code

Reply:
Uncommented the code. But minEigenValue is never used, my suggestion is that maybe we can add a test like "EXPECT_DOUBLES_EQUAL(0, minEigenValue, 1e-11);" to make sure both the eigen value computed by Spectra and Eigen are in the same range?